torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2412,6 +2412,11 @@ def remove_fx_node(self, node: Optional[torch.fx.Node]) -> None:
             self.name_to_node.pop(node.name)
             self.graph.erase_node(node)
 
+    def add_fx_node_metadata(self, node: torch.fx.Node) -> None:
+        from torch._dynamo.utils import get_current_node
+        node.meta["event"] = self.last_event_index()
```

Comment:
This is too generic. Meta dict is used by everyone. Give a more detailed string key